ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -244,6 +248,7 @@ def get_workflow(dict_configs, cdump='gdas'):
     do_gempak = base.get('DO_GEMPAK', 'NO').upper()
     do_awips = base.get('DO_AWIPS', 'NO').upper()
     do_wafs = base.get('WAFSF', 'NO').upper()
+    do_vrfy = base.get('DO_VRFY', 'NO').upper()
```

Comment:
I don't think `DO_VRFY` is an entry in `config.base`.  By setting this to `NO` as default here, it is effectively turning off verification on all machines.

Reply:
Modified to default to YES. User can set it to NO in config.base

Reply:
There is no entry in `config.base` for `DO_VRFY`.
Please add in the appropriate location in `config.base.emc.dyn` and `config.base.nco.static`.

Reply:
Done.